src/context.ts

```diff
@@ -1,4 +1,5 @@
-import React, { createContext, useContext } from 'react';
+import type React from 'react';
```

Comment:
Is it possible to have `type` import in the end of/after all imports?

Reply:
This can be done using another eslint plugin. I can do it in this PR, but I think it would be better to create a separate PR for this 🙂

Reply:
@IvanIhnatsiuk let's do in this, since anyway we are changing import style :)